### PR TITLE
Elaborated on storing of files

### PR DESCRIPTION
**Pull request by alina-tincas**
- specifically on the path and storing

### CODE REVIEW

**Review by ronaldbarendse — APPROVED** [review]

File: 11/umbraco-forms/editor/attaching-workflows/workflow-types.md [hunk 1]
```diff
@@ -45,6 +45,12 @@ Saves the result of the Form as an XML file by using XSLT. The following configu
 * File extension (required)
 * XsltFile - used to transform the XML
 
+The path needs to point to a folder, not a file name. The files are then stored locally, and relative paths are resolved to the content root.
+
+{% hint style="info" %}
+If you store the files within the wwwroot or App_Plugins folders, this will make the files publicly available by default.
```
Comment (sofietoft) [inline 11/umbraco-forms/editor/attaching-workflows/workflow-types.md:51]:
```suggestion
When storing the files within the `wwwroot` or `App_Plugins` folders, the files will be publicly available by default.
```

**Review by sofietoft — COMMENTED** [review]
Thanks for the PR @alina-tincas !

I've made a suggestion to change the wording slightly.
Let me know what you think!

File: 11/umbraco-forms/editor/attaching-workflows/workflow-types.md (hunk 1 above)
Comment (alina-tincas) [reply 11/umbraco-forms/editor/attaching-workflows/workflow-types.md:51]: Looks good :)

**Comment by sofietoft** [comment]
Should this be added to the the v10 version of the article as well?

**Comment by alina-tincas** [comment]
> Should this be added to the the v10 version of the article as well?

I believe so. Maybe @ronaldbarendse can confirm just to be sure? :)

**Comment by sofietoft** [comment]
Hi @alina-tincas !

Just following up here - I'll add the note to the v10 version as well, and merge this one.

**Pull request merged**